incubator/etcd/templates/statefulset.yaml

```diff
@@ -28,147 +28,224 @@ spec:
           name: peer
         - containerPort: {{ .Values.ClientPort }}
           name: client
+        #readinessProbe:
+        #  ${PROTO}Get:
+        #    path: /metrics
+        #    port: {{ .Values.ClientPort }}
         resources:
           requests:
             cpu: "{{ .Values.Cpu }}"
             memory: "{{ .Values.Memory }}"
         env:
         - name: INITIAL_CLUSTER_SIZE
-          value: {{ default 3 .Values.Replicas | quote }}
+          value: {{ default 3 .Values.InitialSize | quote }}
```

Comment:
Why do we need an InitialSize var? Why can't we just continue to use the existing var Replicas i.e. current size

Reply:
Already removed.